spring-core/src/main/java/org/springframework/core/MethodParameter.java

```diff
@@ -329,6 +338,19 @@ public MethodParameter nestedIfOptional() {
 		return (isOptional() ? nested() : this);
 	}
 
+	/**
+	 * Return whether this method parameter is declared as a Kotlin
+	 * nullable.
+	 * @since 4.4
+	 */
+	public boolean isKotlinNullable() {
```

Comment:
Maybe `isNullable()`would be less specific and let the door opened to supporting this is other languages or with specific annotations